README.md

```diff
@@ -228,7 +227,7 @@ Matrix.calculate({
 });
 ```
 
-Or return elevation data from a geojson line:
+Or return elevation data from a geoJson line:
```

Comment:
if you want to use capital letters, the correct notation for this is `GeoJSON` :D